packages/database/schema.prisma

```diff
@@ -500,3 +500,12 @@ model User {
   /// [UserNotificationSettings]
   notificationSettings      Json             @default("{}")
 }
+
+model UrlShortener {
+  id        String   @id @default(cuid())
+  createdAt DateTime @default(now()) @map(name: "created_at")
+  updatedAt DateTime @updatedAt @map(name: "updated_at")
+  fullUrl   String
+  shortUrl  String
+  hits      Int      @default(0)
```

Comment:
Can you please remove hits and corresponding service logic, as of now we don't need that count 😊

Reply:
Sure, I removed the `hits` from prisma schema.